ivadomed/loader/bids_dataframe.py

```diff
@@ -1,11 +1,35 @@
 import copy
 import itertools
 import os
-
 import bids as pybids
 import pandas as pd
 from loguru import logger
+from typing import List
+from ivadomed.keywords import LoaderParamsKW, ROIParamsKW, ContrastParamsKW, BidsDataFrameKW
 from pathlib import Path
+import json
+
+def write_derivatives_dataset_description(path_data: str):
+    """Writes default dataset_description.json file if not found in path_data/derivatives folder
+
+    Args:
+        path_data (str): Path of the data
+
+    Returns:
+
+    """
+    filename = "dataset_description"
+    # Convert path_data to Path object for cross platform compatibility
+    fname_data = Path(path_data)
+    deriv_desc_file = str(fname_data / "derivatives" / f"{filename}.json")
+    label_desc_file = str(fname_data / "derivatives" / "labels" / f"{filename}.json")
+
+    # need to write default dataset_description.json file if not found
+    if not os.path.isfile(deriv_desc_file) and not os.path.isfile(label_desc_file):
```

Comment:
Maybe convert `deriv_desc_file` and `label_desc_file` to be the `Path` objects and use `Path.is_file()` to check if they are exist or not.

Reply:
Yup. Great observation. Converted to the more common path approach.